herddb-core/src/test/java/herddb/core/SimpleOperatorsTest.java

```diff
@@ -294,28 +292,14 @@ public void simpleArithmeticOperationsTest() throws Exception {
             }
 
             // In expressions
-            // Warning: jsqlParser doesn't handle this kind of expressions in select clause
             try (DataScanner scan1 = scan(manager, "SELECT * FROM tblspace1.tsql WHERE '1' in (1,2,3)", Collections.emptyList())) {
```

Comment:
@amitvc  this kind of query is not useful in real life.
We can change the behaviour